components/script/dom/readablestreambyobrequest.rs

```diff
@@ -3,37 +3,72 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 use dom_struct::dom_struct;
+use js::gc::CustomAutoRooterGuard;
+use js::jsapi::Heap;
+use js::typedarray::{ArrayBufferView, ArrayBufferViewU8};
 
+use super::bindings::buffer_source::{BufferSource, HeapBufferSource};
 use crate::dom::bindings::codegen::Bindings::ReadableStreamBYOBRequestBinding::ReadableStreamBYOBRequestMethods;
 use crate::dom::bindings::import::module::{Error, Fallible};
 use crate::dom::bindings::reflector::Reflector;
+use crate::dom::bindings::root::MutNullableDom;
+use crate::dom::readablebytestreamcontroller::ReadableByteStreamController;
 use crate::script_runtime::JSContext as SafeJSContext;
 
 /// <https://streams.spec.whatwg.org/#readablestreambyobrequest>
 #[dom_struct]
 pub(crate) struct ReadableStreamBYOBRequest {
     reflector_: Reflector,
+    controller: MutNullableDom<ReadableByteStreamController>,
+    #[ignore_malloc_size_of = "mozjs"]
+    view: HeapBufferSource<ArrayBufferViewU8>,
 }
 
 impl ReadableStreamBYOBRequestMethods<crate::DomTypeHolder> for ReadableStreamBYOBRequest {
     /// <https://streams.spec.whatwg.org/#rs-byob-request-view>
     fn GetView(&self, _cx: SafeJSContext) -> Option<js::typedarray::ArrayBufferView> {
-        // TODO
-        None
+        self.view.buffer_to_option()
```

Comment:
// Return [this](https://webidl.spec.whatwg.org/#this).[[[view]]](https://streams.spec.whatwg.org/#readablestreambyobrequest-view).

Reply:
I meant that the comment can be added for the step.